### ISSUE TITLE
Installing bcpandas for use in Shiny App using R Package Reticulate

### ISSUE THREAD

**Issue by cjwoodfield** [yehoshuadimarsky/bcpandas#15]
Hello!

Apologies if this isnt exactly the correct forum for this but at my wits end would be an understatement

My colleague has written a great function using bcpandas - the problem is installing it so that it's recognised and can be used by the R Reticulate package

After a fresh Install of miniconda the packages we need to run his code arent available, so I install them...
```
conda_install(envname = "C:\\Users\\xxxxx\\AppData\\Local\\r-miniconda\\envs\\r-reticulate", packages="pandas")
conda_install(envname = "C:\\Users\\xxxxx\\AppData\\Local\\r-miniconda\\envs\\r-reticulate", packages="pyodbc")
conda_install(envname = "C:\\Users\\xxxxx\\AppData\\Local\\r-miniconda\\envs\\r-reticulate", packages="bcpandas")
```
After installation however - bcpandas is not available even though I can see it has been installed into the site packages folder.

```
> py_module_available("numpy")
[1] TRUE
> py_module_available("glob")
[1] TRUE
> py_module_available("os")
[1] TRUE
> py_module_available("numpy") 
[1] TRUE
> py_module_available("subprocess")
[1] TRUE
> py_module_available("pyodbc") 
[1] TRUE
> py_module_available("pandas")
[1] TRUE
> py_module_available("bcpandas")
[1] FALSE
```

Do you know why this might be the case?

**Comment by cjwoodfield** [comment]
here's the more detailed error message...

```
> reticulate::py_run_string("import bcpandas")
Error in py_run_string_impl(code, local, convert) : 
  OSError: [WinError 6] The handle is invalid

Detailed traceback: 
  File "<string>", line 1, in <module>
  File "C:\Users\xxxxxx\Documents\R\win-library\4.0\reticulate\python\rpytools\loader.py", line 31, in _import_hook
    level=level
  File "C:\Users\xxxxxx\AppData\Local\R-MINI~1\envs\R-RETI~1\lib\site-packages\bcpandas\__init__.py", line 14, in <module>
    run(["bcp", "-v"], stdout=DEVNULL, stderr=DEVNULL)
  File "C:\Users\xxxxxx\AppData\Local\R-MINI~1\envs\R-RETI~1\lib\subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Users\xxxxxx\AppData\Local\R-MINI~1\envs\R-RETI~1\lib\subprocess.py", line 687, in __init__
    errread, errwrite) = self._get_handles(stdin, stdout, stderr)
  File "C:\Users\xxxxxx\AppData\Local\R-MINI~1\envs\R-RETI~1\lib\subprocess.py", line 910, in _get_handles
    p2cread = _winapi.GetStdHandle(_winapi.STD_INPUT_HANDLE)
```

**Comment by yehoshuadimarsky** [comment]
Fixed in #16

**Comment by cjwoodfield** [comment]
Thanks for your fix the other day! bcpandas (V1.0.1) now imports fine via the reticulate package into our R App & our function starts to run.

We get an error however when using the to_sql function when we reach line 158...

```
to_sql(dfgroups[dfg], tablename, sql_config, index=True, if_exists='append')
```
The error message (below) shows the same 'handle is invalid' message as we received the other day - so Im wondering if these issues could be linked?
```
Error: OSError: [WinError 6] The handle is invalid Detailed traceback: File "",
line 158, in F_Upload_P File "C:\Users\woodfieldc\AppData\Local\r-miniconda\envs\r-reticulate\lib\site-packages\bcpandas\main.py", 
line 348, in to_sql bcp_path=bcp_path, File "C:\Users\woodfieldc\AppData\Local\r-miniconda\envs\r-reticulate\lib\site-packages\bcpandas\utils.py", 
line 114, in bcp ret_code = run_cmd(bcp_command) File "C:\Users\woodfieldc\AppData\Local\r-miniconda\envs\r-reticulate\lib\site-packages\bcpandas\utils.py", 
line 231, in run_cmd proc = Popen(cmd, stdout=PIPE, stderr=PIPE, encoding="utf-8", errors="utf-8", shell=with_shell,) File "C:\Users\woodfieldc\AppData\Local\r-miniconda\envs\r-reticulate\lib\subprocess.py", 
line 687, in __init__ errread, errwrite) = self._get_handles(stdin, stdout, stderr) File "C:\Users\woodfieldc\AppData\Local\r-miniconda\envs\r-reticulate\lib\subprocess.py", 
line 910, in _get_handles p2cread = _winapi.GetStdHandle(_winapi.STD_INPUT_HANDLE)
```
